storage/mongo/utils.js

```diff
@@ -1,27 +1,28 @@
 var MongoClient = require('mongodb').MongoClient;
-require('when.js');
+var when = require('when');
 
 var mydb;
 
 module.exports = {
   connectToServer: function (config) {
     return when.promise(function (resolve, reject) {
-      if (!config.host || !config.port || !config.name) {
+      if (config.db.host == null || config.db.port == null || config.db.name == null) {
```

Comment:
No need to check them with null only they could be `undefined`, `""`
for example if i am checking for a variable x 
then i can do 
if(x){
}
x = undefined, null , empty string will be treated as false over here.